devlog/_plan/260901_merge_train_round3/060_wp7_websocket_refresh_flake.md

```
@@ -0,0 +1,187 @@
+src/codex/quota.ts:457
+  updatedAt: Date.now(),
+```
+
+The fixture calls `updateAccountQuota("pool-a", 10, 5)` at `:2242`, **before**
+`Date.now` is faked — so `updatedAt` is stamped with the **real** clock, 2026-09-01.
+
+Except that table was a prediction, and measuring it refuted the interesting half.
+
+## Correction: the prime is ALWAYS stale, before and after the fix
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)